boat_tutorials/conceptual/target_strength.md

```diff
@@ -1,3 +1,62 @@
-# Target Strength
+# Target strength
 
-Coming soon!
+
+## Introduction
+- intuitively, what is target strength (TS)? 
+    - how much sound energy bounces off (scatters) from a target? 
+absorption
+    - more generally: scattering strength?
+- when/how is it used?
+    - characterize scatterers or scattering sources independent of the complications of the transmit signals ([connec to SL]()) and the propagation environment ([connect to TL]())
+
+
+## Setup and terminology
+- monostatic vs bistatic ([deeper topic]())
+- differential scattering cross section --> TS ([connect to decibel page]())
+- total scattering cross section
+- extinction cross section (total power removed from the incident sound):
+    - scattering cross section + absorption cross section
+
+## Discrete scatterers
+- what can we infer using these models?
+    - fish/zooplankton abundance
+    - density of suspended sediment ([deeper topic]())
+- point scatterer
+- spherical cows
+    - rigid sphere
+    - bubble
+    - elastic sphere ([deeper topic]())
+- non-spherical cows
+    - bring in orientation ([connect to SL/beampattern]()) and frequency dependency ([connect to spectrum]())
+    - types:
+        - air inclusion (fish swimbladder)
+        - soft body (squid, krill)
+        - elastic structure (pteropod)
```

Comment:
- show TS spectra for different spheres
- solve (?) rigid  sphere and bubble
- talk about frequency dependency using bubble example
- orientation dependency for non-spherical examples